gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/Query.java

```diff
@@ -30,7 +30,7 @@
 
 /**
  * A Google Cloud Datastore query.
- * For usage examples see {@link GqlQuery} and {@link StructuredQuery}.
+ * For usage example see {@link GqlQuery} and {@link StructuredQuery}.
```

Comment:
Wait, I didn't mean to change anything there! I think the IDE went a little crazy with the "helpful" renaming as I moved Stat. I'll revert that.